src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,56 @@
+.stopwatch {
+  position: relative;
+  width: $stopwatch-size;
+  height: $stopwatch-size;
+
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+
+  &__minutes-hand {
+    position: absolute;
+    bottom: 50%;
+    left: center-stopwatch-elements($minutes-hand-width);
+
+    height: $minutes-hand-hight;
+    width: $minutes-hand-width;
+
+    background-color: #0700ff;
+
+    transform-origin: bottom;
+    animation: hand-move 3600s steps(60) infinite;
+  }
+
+  &__seconds-hand {
+    position: absolute;
+    bottom: 50%;
+    left: center-stopwatch-elements($seconds-hand-width);
+
+    height: $seconds-hand-hight;
+    width: $seconds-hand-width;
+
+    background-color: #2c8000;
+
+    transform-origin: bottom;
+    animation: hand-move 60s linear infinite;
+  }
+
+  &__center {
+    position: absolute;
+    top: center-stopwatch-elements($center-size);
+    left: center-stopwatch-elements($center-size);
```

Comment:
The value 'center-stopwatch-elements' is not a valid CSS value. If it's a function or a mixin, ensure that it is defined and used correctly.